src/MassTransit/Configuration/Registration/DefaultBusInstance.cs

```diff
@@ -17,5 +20,19 @@ public DefaultBusInstance(IBusControl busControl)
         public IBusControl BusControl { get; }
 
         public IHostConfiguration HostConfiguration => default;
+
+        public void Connect(IBusAttachment attachment)
```

Comment:
I'm guessing this is just never called, but if it is, should we throw an exception guiding them to the documentation on how to use it?